civicrm.php

```diff
@@ -507,6 +519,9 @@ public function register_hooks_admin() {
     // set page title
     add_filter( 'admin_title', array( $this, 'set_admin_title' ) );
 
+    // print CiviCRM's header
+    add_action('admin_head', array( $this, 'wp_head' ), 50);
```

Comment:
Yes, I saw that you'd relocated it. I can't give an opinion on it until I've had a chance to test. That may not be until next week, I'm afraid. Deadline looming *sigh*